mne/viz/topo.py

```diff
@@ -152,6 +153,11 @@ def _plot_topo(info, times, show_func, click_func=None, layout=None,
     """Helper function to plot on sensor layout."""
     import matplotlib.pyplot as plt
 
+    if layout.kind == 'custom':
+        layout = copy(layout)
+        for i_dim in layout.pos.T[:2]:
+            i_dim -= i_dim.min(0)
+            i_dim /= (i_dim.max(0) - i_dim.min(0))
```

Comment:
What do you think of this? This basically normalizes the xy-coords for plotting the channels in relation to each other (for topo). It does not show the locations in relation to the head, but this function never claims to do so anyway.